shortbread-website/content/make-vectortiles/planetiler.md

```diff
@@ -17,6 +17,7 @@ Shortbread Tiles can be created with [Planetiler](https://github.com/onthegomap/
 ### Quickstart (Docker)
 
 ```bash
+mkdir data
```

Comment:
That seems unrelated to who the licensor is.

Reply:
Ah, sorry, didn't spot the
> and fix directory permission in planetiler tutorial.

in the PR description. Though, while I don't know this project's contribution conventions, it's usually a good idea to submit logically independent changes separately, rather than lumping them together in a single PR. :slightly_smiling_face: